crm_claim/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "name": "Claims Management",
     "version": "14.0.1.0.0",
     "category": "Customer Relationship Management",
-    "author": "Odoo S.A., Tecnativa, Odoo Community",
+    "author": "Odoo Community Association (OCA)",
```

Comment:
Why authors are removed?

Reply:
@redapureskill Please restore Odoo SA and Tecnativa as authors. Then I could merge this.

Reply:
@NL66278 This changes among others are done in this #391

Reply:
@Daniel-CA I do not quite understand what you mean with the reference to the other PR. We should not just delete authors...

Reply:
Please review the other PR as superseding this one.